docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -277,6 +222,8 @@ const PostType = () => {
 export default connect(PostType);
 ````
 
+---
+
 ## More things added
 
 ### `actions.source.updateState`
```

Comment:
Change:
"Action that replaces the value of `state.router.state` with the give object."
to:
"Action that replaces the value of `state.router.state` with the given object."
below.